File: CPP/constructor_employee.cpp
#include<iostream>
using namespace std;
class employee
{
	public:
		int empno;
		string empname;
		employee(int eno,string en)
{
	empno=eno;
	empname=en;
}
void display()
{
	cout<<"empno"<<empno<<endl;
	cout<<"empname"<<empname<<endl;
}
};
int main()
{
	int eno;
	string en;
	employee emp1(11,"mayuri");
	emp1.display();
}

File: C#/constructor_employee.cs
using System;
namespace program
{
    class Employee
    {
        int empno;
        string empname;
        string designation;
        int salary;

        public Employee()
        {
            empno = 1;
            empname = "mayuri";
            designation = "manager";
            salary = 50000;
        }
        public void display()
        {
            Console.WriteLine("emp no : " + empno);
            Console.WriteLine("emp name : " + empname);
            Console.WriteLine("designation : " + designation);
            Console.WriteLine("salary : " + salary);
        }
    }
    class program
    {
        public void Main(string[] args)
        {
            Employee emp = new Employee();
            emp.display();
            Console.ReadKey();
        }
    }
}